kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```
@@ -2594,13 +2594,15 @@ components:
 
     NewSchemaSubject:
       type: object
+      description: should be set for creating new and also updating schema subject
       properties:
         subject:
           type: string
         schema:
           type: string
         schemaType:
           $ref: '#/components/schemas/SchemaType'
+          description: should be set and provide the existing schema type when updating schema subject
```

```suggestion
          description: upon updating a schema, the type of an existing schema can't be changed
```
made it a bit clearer